CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Fix: missing lastSuccess/lastFailure associated to initial notification on subscription creation some times when csub cache is in use (#2974)
+- Add: release_date and doc fields are added to the GET /version output to align with FIWARE scheme (#2971)
```

Comment:
The number of the issue is 2970. Could you change it, please? Sorry, I did a typo on my first comment about this :(

Reply:
No worries, it is my mistake, copy-pasting without thinking :)

Fixed in 957bf89